src/main/java/com/google/events/firebase/remoteconfig/v1/RemoteConfigEventData.java

```diff
@@ -58,11 +58,12 @@
     /**
      * What type of update was made.
      */
-    public Update getUpdateType() { return updateType; }
-    public void setUpdateType(Update value) { this.updateType = value; }
+    public UpdateType getUpdateType() { return updateType; }
```

Comment:
A number of these changes seem like compatibility breaks. Is the plan to roll a new major?